tokio/src/runtime/builder.rs

```diff
@@ -375,6 +381,26 @@ impl Builder {
             blocking_pool,
         })
     }
+
+    /// Sets a custom timeout for a thread in the `BlockingPool`.
```

Comment:
nit: `BlockingPool` isn't a type in the public API, so I would probably change this to 
```suggestion
    /// Sets a custom timeout for a thread in the blocking pool.
```

Reply:
Also, it would be nice if the docs for this method said more about what this actually means — what does this timeout control? — and maybe mentions why you might want to override the default value?